course02/homework00/skeleton/vector.h

```diff
@@ -1,8 +1,45 @@
 #pragma once
 
+#include <array>
+
 class VectorInt
 {
-	// TODO
+	
+public:
+	int32_t size()
```

Comment:
dont use fixed width int when you dont need to - they are less portable, might be less performant, and it's odd to add restrictions without good reasons